poo/herencia/telefonoCamara/index.ts

```diff
@@ -0,0 +1,48 @@
+class Telefono{
+    private onOf: boolean;
```

Comment:
fijate que seria Off en vez de of para que este bien en ingles

Reply:
Uff siiii jaja